sequence_tagging_for_ner/train.py

```diff
@@ -66,7 +68,7 @@ def main(train_data_file, test_data_file, vocab_file, target_file, emb_file,
 
     inference_program = fluid.default_main_program().clone()
     with fluid.program_guard(inference_program):
-        test_target = chunk_evaluator.metrics + chunk_evaluator.states
+        test_target = chunk_evaluator.metrics
```

Comment:
于老师， 我看咱们警告是说 这个evaluator.py 不用啦？   这一行需要一块改掉么？

	/usr/local/lib/python2.7/dist-packages/paddle/fluid/evaluator.py:69: Warning: The ChunkEvaluator is deprecated, because maintain a modified program inside evaluator cause bug easily, please use fluid.metrics.ChunkEvaluator instead.